docs/versioned_docs/version-4.0.x/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/additional-information/legacy-upgrade.md

```diff
@@ -35,11 +35,11 @@ As compared to Replicated PV Mayastor 1.0, the Replicated PV Mayastor 2.x featur
 **Commands**
 
 ```
-kubectl delete deploy core-agents -n mayastor
-kubectl delete deploy csi-controller -n mayastor
-kubectl delete deploy msp-operator -n mayastor
-kubectl delete deploy rest -n mayastor
-kubectl delete ds mayastor-csi -n mayastor
+kubectl delete deploy core-agents -n openebs
+kubectl delete deploy csi-controller -n openebs
+kubectl delete deploy msp-operator -n openebs
+kubectl delete deploy rest -n openebs
+kubectl delete ds mayastor-csi -n openebs
```

Comment:
Legacy was mayastor..
```suggestion
kubectl delete deploy core-agents -n mayastor
kubectl delete deploy csi-controller -n mayastor
kubectl delete deploy msp-operator -n mayastor
kubectl delete deploy rest -n mayastor
kubectl delete ds mayastor-csi -n mayastor
```

Reply:
Maybe we should not modify this file at all, wdyt?

Reply:
Done.